modules/enumeratum/README.md

```diff
@@ -42,7 +47,6 @@ val conf = parseString("""{
   end: SHOUT_GOOD_BYE
 }""")
 loadConfig[GreetingConf](conf)
```

Comment:
The enumeratum module doesn't build with tut right now -- it complains about a missing import. I'm guessing that's why there's no output.